### ISSUE TITLE
Concentric 3D should be removed

### ISSUE THREAD

**Issue by BagelOrb** [Ultimaker/Cura#3774]
Because of https://github.com/Ultimaker/Cura/issues/3408 and new insights I think the Concentric 3D infill type should be removed.

Concentric was already not the best infill type with lots of situations in which it failed.
Concentric 3D has even more failing situations.
Even when the model has the same layers throughout it can fail.

Failing situations
- the outlines of the model change too much, causing overhang violations
- the shape of the layers is such that there are parts with a width different from an even number of multiples of the infill line distance

The pattern fails often and it is rather difficult to explain the above constraints to the user.

Here's a simple cube which fails.
![image](https://user-images.githubusercontent.com/8895761/39627417-251e3b60-4fa6-11e8-8ed5-01d8098f0dc0.png)

**Comment by BagelOrb** [comment]
Results for different model widths, using an Infill Line Distance of 5:

![image](https://user-images.githubusercontent.com/8895761/39627734-2296e364-4fa7-11e8-9223-2132c7edd090.png)
![image](https://user-images.githubusercontent.com/8895761/39627754-2e470aa4-4fa7-11e8-9039-72be7de8b7ef.png)
![image](https://user-images.githubusercontent.com/8895761/39627840-6dfe472a-4fa7-11e8-8fed-0d22401694f0.png)


exact multiple of line distance:
![image](https://user-images.githubusercontent.com/8895761/39627783-42429f32-4fa7-11e8-96eb-d5dba81e1d11.png)

**Comment by DDDirk** [comment]
It just so happens that I used concentric and concentric3D for the first time yesterday, with a model that is basically a cylinder with a slowly changing diameter. Not surprisingly, this worked just fine (just looking at the rendered gcode). But then I also made a version with a ridiculous large number of wall lines (19 of them), which does almost the same. The main difference is that that prints the walls from the inside out, whereas the concentric versions do this from the outside in (which also results in an extra long travel move at the end because it does inner before outer walls).

I have used the alternative with many wall lines a few times before, and I can't remember having any problems with that. So might this be presented as an alternative? Or might it be a th basis for a better approach to do concentric?

**Comment by DDDirk** [comment]
I'm not sure if you want the model or the gcode, so I'll upload both.
[tensile_spiral_98.stl.zip](https://github.com/Ultimaker/Cura/files/1975025/tensile_spiral_98.stl.zip)
[tensile_spiral_98_Cura_concentric3D.gcode.zip](https://github.com/Ultimaker/Cura/files/1975026/tensile_spiral_98_Cura_concentric3D.gcode.zip)

**Comment by BagelOrb** [comment]
I was asking for the project file.
File > Save Project.

**Comment by DDDirk** [comment]
Ah, I didn't know about that. Here it is:
[tensile_spiral_98.zip](https://github.com/Ultimaker/Cura/files/1976605/tensile_spiral_98.zip)

**Comment by BagelOrb** [comment]
I don't know from which version your project file is, it has version_number unknown and I cannot open it as a project in Cura 3.3.1...

This is the problem I get with your model when I set the infill to Concentric 3D:
That middle piece is unsuported.
![image](https://user-images.githubusercontent.com/8895761/39663199-22e79804-506f-11e8-80d6-c31b3c2633bf.png)

**Comment by DDDirk** [comment]
I used Cura 3.0.0.
Now I see the problem. It occurs when you don't use 100% infill, as I did. And now I also understand the difference between concentric and concentric3D. Like I said, this was the first time I used it.

**Comment by Ghostkeeper** [comment]
This is implemented by https://github.com/Ultimaker/Cura/pull/3920 and https://github.com/Ultimaker/CuraEngine/pull/794.

**Comment by ekohlwey** [comment]
So I upgraded to cura 3.5.0 today and noticed 3d concentric was gone. This is problematic for me. I like using 3d concentric because it reduces ringing in the parts. My main purpose in 3d printing is producing precision parts where reduction in strength is ok but volumetric distortions caused by ringing are quite unacceptable. Most of the parts I make are rectilinear and avoid the problematic cases you outline. While It is true that the edge cases you propose are hard to explain to users, it is also true that ringing as a concept is hard to explain to most users and this is a nice infill pattern to address it in software, compared to rather extensive hardware overhauls to remove the effects.

I would really like it if this was added back in, as I have a number of parts that become quite spongey if I do not have this turned on, or which ring unacceptably if using another infill. I get that there's lots of corner cases, but this was a pretty key feature for me in Cura because of the specific impact of reducing ringing in the model while providing some radial strength.
